src/Containers/App/App.js

```diff
@@ -1,15 +1,51 @@
 import React, { Component } from 'react';
+import { NavLink, Route } from 'react-router-dom';
+import NotesSection from '../NotesSection/NotesSection';
+import NoteForm from '../NoteForm/NoteForm';
+import NotFound from '../../Components/NotFound/NotFound';
+import { connect } from 'react-redux';
+import { setNotes, setLoading } from '../../actions';
 
 class App extends Component {
+
+  componentDidMount() {
+    const { setNotes, setLoading } = this.props;
+    const notes = [];
+    setNotes(notes);
+    setLoading(false);
+  }
+
   render() {
+    const { notes } = this.props;
     return (
       <div>
         <header>
-          <h1>Listy</h1>
+          <NavLink id='title' to='/'>
+            <h1>Listy</h1>
+          </NavLink>
+          <NavLink id='add-note-link' to='/new-note'>
+            <i className="fas fa-plus-circle"></i>
+          </NavLink>
         </header>
+        <Route exact path='/' component={NotesSection} />
```

Comment:
Maybe we could put in a Switch component to wrap our Routes so that only one Route can be selected at a time?

Reply:
Agree, that will be good to add.